Spore ModAPI/Spore ModAPI.vcxproj

```diff
@@ -1078,6 +1078,7 @@
     <ClCompile Include="SourceCode\Object.cpp" />
     <ClCompile Include="SourceCode\ResourceKey.cpp" />
     <ClCompile Include="SourceCode\UTFWin\WindowManager.cpp" />
+    <ClCompile Include="Spore\UTFWin\IButton.cpp" />
```

Comment:
I think it is not worth adding a new source file for so little data (in fact, i think I added too many on the past...). It increases complexity and compilation time. Just put the new methods in some existing UTFWin .cpp